x/liquidstaking/keeper/liquidstaking.go

```diff
@@ -600,7 +602,7 @@ func (k Keeper) DoLiquidStake(ctx sdk.Context, msg *types.MsgLiquidStake) (
 		return
 	}
 
-	nas := k.GetNetAmountState(ctx)
+	mintRate := nas.CalcMintRate()
```

Comment:
I think `MintRate` is already calculated in `nas.MintRate`, is there another reason to do it?

Reply:
Thanks for the comment. I should fix that part too.

Reply:
@dongsam 

Fixed it including removing `keeper.GetAvailableChunkSlots` also. https://github.com/b-harvest/Canto/pull/18/commits/0dd25100f7b881a83863b8c99c4b47dd8a2d8ba8